.travis.yml

```diff
@@ -30,11 +29,10 @@ env:
   matrix:
     - DRIVER=ruby      REDIS_BRANCH=3.0
     - DRIVER=ruby      REDIS_BRANCH=3.2
-    - DRIVER=hiredis   REDIS_BRANCH=3.0
-    - DRIVER=hiredis   REDIS_BRANCH=3.2
-    - DRIVER=synchrony REDIS_BRANCH=3.0
-    - DRIVER=synchrony REDIS_BRANCH=3.2
     - DRIVER=ruby      REDIS_BRANCH=4.0
+    - DRIVER=ruby      REDIS_BRANCH=5.0
+    - DRIVER=hiredis   REDIS_BRANCH=5.0
+    - DRIVER=synchrony REDIS_BRANCH=5.0
```

Comment:
The idea here is to test all the versions of the server with the ruby driver, but only the most recent one with the alternative drivers. I believe it's a good compromise between coverage and matrix size.